fastlane/Fastfile

```diff
@@ -459,21 +536,63 @@ platform :ios do
   #
   desc 'Creates a new hotfix branch for the given `version:x.y.z`. The branch will be cut from the `x.y` tag.'
   lane :new_hotfix_release do |options|
-    prev_ver = ios_hotfix_prechecks(options)
-    ios_bump_version_hotfix(previous_version: prev_ver, version: options[:version], skip_deliver: true)
+    new_version = options[:version] || UI.input('Version number for the new hotfix?')
+    hotfix_build_code = hotfix_build_code(release_version: new_version)
+
+    # Parse the provided version into an AppVersion object
+    parsed_version = VERSION_FORMATTER.parse(new_version)
+    previous_version = VERSION_FORMATTER.release_version(VERSION_CALCULATOR.previous_patch_version(version: parsed_version))
+
+    # Check versions
+    message = <<-MESSAGE
+
+      Current release version: #{current_release_version}
+      New hotfix version: #{new_version}
+
+      Current build code: #{current_build_code}
+      New build code: #{hotfix_build_code}
+
+      Branching from tag: #{previous_version}
+
+    MESSAGE
+
+     UI.important(message)
+     UI.user_error!('Aborted by user request') if !options[:skip_confirm] && !UI.confirm('Do you want to continue?')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/c13a215366b1c2abb0171201a62dc6a39267e30b/fastlane/Fastfile#L560"></span>Inconsistent indentation detected.</td>
    </tr>
  </tbody>
</table>

Reply:
Fixed in https://github.com/Automattic/pocket-casts-ios/pull/1137/commits/d951f537ecef9223930535e1d1bc9bf9a14b97f8